featured-speakers.js

```diff
@@ -0,0 +1,81 @@
+    speaker_title: '<small>Berkman Professor of Entrepreneurial legal studies of Havard Law School</small>',
+    description: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+  },
+  {
+    feature_img: './images/speaker_02.png',
+    official_name: 'Yochai Benkler',
+    nickname: 'Yochai',
+    speaker_title: '<small>Berkman Professor of Entrepreneurial legal studies of Havard Law School</small>',
+    description: 'Kilnam Chon helped bring the internet to Asia and is an outspoken advocate for the open web and digital com-mons. In 2012. he was inducted into the inaugural class of the Internet Society’s (ISOC) Internet Hall of Fame',
+  },
+  {
+    feature_img: './images/speaker_03.png',
+    official_name: 'Yochai Benkler',
+    nickname: 'Yochai',
+    speaker_title: '<small>Berkman Professor of Entrepreneurial legal studies of Havard Law School</small>',
+    description: 'As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.',
+  },
+  {
+    feature_img: './images/speaker_04.png',
+    official_name: 'Yochai Benkler',
+    nickname: 'Yochai',
+    speaker_title: '<small>Berkman Professor of Entrepreneurial legal studies of Havard Law School</small>',
+    description: 'European ingetration, political democracy and participation of youth through online as her major condern, Reda’s report outlining potential changes to EU copyright law was approved by the Parliament in July ',
+  },
+  {
+    feature_img: './images/speaker_05.png',
+    official_name: 'Yochai Benkler',
+    nickname: 'Yochai',
+    speaker_title: '<small>Berkman Professor of Entrepreneurial legal studies of Havard Law School</small>',
+    description: 'Lila Tretikov is the Executive of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely available in 290 languag-es and used by nearly half a billion people around the world every month.',
+  },
+  {
+    feature_img: './images/speaker_06.png',
+    official_name: 'Yochai Benkler',
+    nickname: 'Yochai',
+    speaker_title: '<small>Berkman Professor of Entrepreneurial legal studies of Havard Law School</small>',
+    description: 'Ryan had been leading open-source projects at the Mozilla Foundation such as the open-source move-ment',
+  },
+];
+const feature_speakers = document.getElementById('feature');
+
+const createElementAndAppendToParent = (newElement, className, id, parent, HTMLtext) => {
+  const element = document.createElement(`${newElement}`);
+  element.className = `${className}`;
+  element.id = `${id}`;
+  element.innerHTML = `${HTMLtext}`;
+  parent.appendChild(element);
+  return element;
+};
+
+// eslint-disable-next-line no-undef
+const speaker_heading = createElementAndAppendToParent('h2', 'featured_speakers', 'featured-speaker', feature_speakers, 'Featured Speakers');
```

Comment:
- [ ] As suggested by the previous reviewer,  I think it would be better if you follow the design provided on [Behance.net](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as closely as possible. The featured speaker's title should have a reddish #EC5242 divider at the bottom as captured on the Behance design, please rectify this 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/226113858-1fbb6b3e-5781-4525-9447-b2c0b42aeaba.png)

design guidelines 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/226113829-7f9c9cc7-8d2e-4dc2-aade-c1efffb68a76.png)